tests/e2e/e2e_exec_test.go

```diff
@@ -442,6 +442,7 @@ func (s *IntegrationTestSuite) executeDelegate(c *chain, valIdx int, amount, val
 		fmt.Sprintf("--%s=%s", flags.FlagGas, "auto"),
 		fmt.Sprintf("--%s=%s", flags.FlagFees, delegateFees),
 		"--keyring-backend=test",
+		fmt.Sprintf("--%s=%s", flags.FlagGasAdjustment, "1.5"),
```

Comment:
Is this new flag documented somewhere?

Reply:
Yes, without it some of the Txs fail with insufficient fees.